### ISSUE TITLE
How to fix? Invalid certificate

### ISSUE THREAD

**Issue by furzich2015** [ultrafunkamsterdam/undetected-chromedriver#730]
![image](https://user-images.githubusercontent.com/36245898/179527190-7bf56907-111f-4ccd-813f-9a622da279f8.png)

**Comment by sebdelsol** [comment]
You're on the **wrong repo**, 
Please check the existing issues on **[the relevant repo](https://github.com/wkeeling/selenium-wire/issues?q=is%3Aissue+invalid+certificate)**.

**Comment by furzich2015** [comment]
Ohh, thanks. I use undetected-chromedriver and wire from proxy

**Comment by sebdelsol** [comment]
Sure, note that you [**_have other alternatives to proxies_**](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/639#issuecomment-1133890051).

**Comment by furzich2015** [comment]
Yeah, but i have proxy with authentication, and i use wire. But i use wire and show this Invalid certificate. 
When I use just undetected-chromedrive, then everything is fine with the certificate
